### ISSUE TITLE
Main Genre Category & Advanced Tagging

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#280]
- The ability to select a singular "MAIN GENRE" tag (designated as Category) for your roleplay or location, to help filter and prioritize results
- filter and prioritize results considering MAIN GENRE tags before SUB GENRE tags

via thelordgeneral
![image](https://user-images.githubusercontent.com/52435234/77675030-d0df6180-6f62-11ea-822a-6579c3633450.png)

**Comment by martindale** [comment]
This will require a fixed hierarchy, which is added complexity that I'd like to defer until after 1.0 is complete.

**Comment by lostamongtrees** [comment]
Wonderful!
further notes from lordgeneral
> categories should be general and focus on the core theme of the roleplay universe purely to better filter for users and game runners
